content/applications/marketing/email_marketing/analyze_metrics.rst

```diff
@@ -17,84 +20,182 @@ locations.
+A high received rate can indicate the mailing list used is up-to-date, and the sender authentication
+is trusted by email providers.
 
-A low received rate may indicate issues either with the mailing list used for the mailing or with
+A low received rate may indicate issues, either with the mailing list used for the mailing, or with
 the sender authentication. View the :ref:`email-marketing/deliverability-issues` section for more
 information.
 
 Bounced rate
 ------------
 
-The *Bounced* rate measures the percentage of emails that were unsuccessfully delivered, against the
-total number of sent emails.
+This rate measures the percentage of emails that were **unsuccessfully** delivered, and did not
+enter a recipient's inbox, against the total number of sent emails.
 
-A high bounce rate could indicate issues either with the mailing list used for the mailing or with
+A high bounce rate could indicate issues, either with the mailing list used for the mailing, or with
 the sender authentication.
 
-A low bounce rate may indicate that the mailing list used is up to date and the sender
+A low bounce rate may indicate that the mailing list used is up-to-date, and the sender
 authentication is trusted by email providers. View the :ref:`email-marketing/deliverability-issues`
 section for more information.
 
+.. tip::
+   Click on the respective smart buttons to see all the corresponding recipient records that are
+   attributed to each metric. When these filtered records are in view, multiple types of reports can
+   be run for further analysis.
+
+.. _email_marketing/create_reports:
+
+Create metrics reports
+======================
+
+Individual metrics can be analyzed by creating a report. To begin, click on the smart button of the
+desired metric.
+
+Next, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` to the right of the search bar to see
+a drop-down menu of filtering and grouping parameters.
+
+:guilabel:`Filters`, located in the left column of the search options, can be used to keep only the
+results that fit the filter. For example, selecting the *Bounced* filter only shows emails
+that could not be delivered.
+
+:guilabel:`Group By`, found in the middle column, is used to organize the results into groups, and
+can be used with or without filters.
+
+.. note::
+  Setting multiple :guilabel:`Group By` options creates nested groups, according to which option
+  is selected first. For example, selecting :guilabel:`Sent Period`, followed by
+  :menuselection:`Add Custom Group -->`:guilabel:`Responsible`, in the :guilabel:`Group By` column,
+  sorts all results *first* by the sent period, *then* by the team member responsible. This is a
+  useful metric for analyzing who on the team is sending in volume or quantity over a set time
+  period.
+
+  This can be verified by looking at the direction, and order, of the selections in the group tile
+  that appears in the search bar after the selections are made.
```

Comment:
Indent entire text to align with the admonition, also fix menuselection :)
```suggestion
   Setting multiple :guilabel:`Group By` options creates nested groups, according to which option
   is selected first. For example, selecting :guilabel:`Sent Period`, followed by
   :guilabel:`Add Custom Group --> Responsible`, in the :guilabel:`Group By` column, sorts all
   results *first* by the sent period, *then* by the team member responsible. This is a useful
   metric for analyzing who on the team is sending in volume or quantity over a set time period.

   This can be verified by looking at the direction, and order, of the selections in the group tile
   that appears in the search bar after the selections are made.
```